facebook/OpenBIC
# fby3.5: hd: Support IPMB

Summary:
- Support IPMB handler.

Dependency: #377 

Test plan:
- Build code: Pass
- Check IPMB is workable: Pass

LOG:
```
root@bmc-oob:~# bic-util slot1 0x18 0x1
00 80 FF FF 02 BF 15 A0 00 00 00 00 00 00 00
```
